acceptance/configure_director_test.go

```diff
@@ -302,6 +308,131 @@ var _ = Describe("configure-director command", func() {
 			"internet_connected": true,
 			"elb_names": ["my-elb"]
 	  }`))
+		})
 	})
 
+	Context("when specifying a config file", func() {
+		It("configures the BOSH director using the API", func() {
+			configJSON := []byte(`
```

Comment:
Minor quibble. At the acceptance test level, I'd prefer to see something closer to canonical usage as it also serves as executable documentation for the app. For example, the same thing in the `configure_product` case, [here](https://github.com/pivotal-cf/om/blob/master/acceptance/configure_product_test.go#L68-L96).

Just my 2 cents!

Reply:
Honestly, I was following the example used above with the standard flags. Happy to work with someone on that canonical example. I wouldn't want it to stop this PR, though.

Reply:
> I wouldn't want it to stop this PR, though.

Yep, agreed. 

I think we're still in the early days of figuring out how we handle PRs effectively. Hoping for things like this, if the team has opinions they can just fix them up on the merge and then provide the feedback in a way that helps guide future contributions. 

And following existing patterns seems like a good recommendation for contributors, so we should try to keep those updated with our opinions as they change to help guide others.

Reply:
@jtarchie not sure if this was what Dave was hinting at, but could we make this provided config file fixture string YAML rather than JSON? Little easier to read in the test as we don't have to escape stuff and it more closely matches how we expect the command to be used.